src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -13,12 +13,13 @@
     <BlazorWebAssemblyLoadAllGlobalizationData>true</BlazorWebAssemblyLoadAllGlobalizationData>
 
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    
+
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TestTrimmedOrMultithreadingApps)' == 'true'">
     <!-- Avoid spending time brotli compression publish output.-->
     <_BlazorBrotliCompressionLevel>NoCompression</_BlazorBrotliCompressionLevel>
+    <WasmApplicationEnvironmentName>Staging</WasmApplicationEnvironmentName>
```

Comment:
This will affect all our tests. 

Could we minimize the blast radius here? For the most part, we want all our tests to run against `Production` as opposed to a custom environment, and only have 1 (or a handful) of tests run against a custom environment.

My worry is that we lose coverage of the common case.

Reply:
Do you have any suggestion how to accomplish it? IIUC the `BasicTestApp` is built just once for all tests. So the only way that comes to my mind is to add a new test app. Otherwise I can move the test case to runtime repository where we have more freedom in how test apps are built and created in tests.